.github/actions/install-maven/action.yml

```diff
@@ -4,8 +4,8 @@ runs:
   steps:
   - name: Install Maven
     run: |
-      wget --no-check-certificate https://dlcdn.apache.org/maven/maven-3/3.9.10/binaries/apache-maven-3.9.10-bin.tar.gz
-      tar zxf apache-maven-3.9.10-bin.tar.gz
-      echo "export M2_HOME=$(pwd)/apache-maven-3.9.10" >> $GITHUB_ENV
-      echo "$(pwd)/apache-maven-3.9.10/bin" >> $GITHUB_PATH
+      wget --no-check-certificate https://rocksdb-deps.s3.us-west-2.amazonaws.com/maven/maven-3/3.9.11/binaries/apache-maven-3.9.11-bin.tar.gz
```

Comment:
I think it would be better to use the Apache Archive, this should remain stable and not break - https://archive.apache.org/dist/maven/maven-3/3.9.11/binaries/apache-maven-3.9.11-bin.tar.gz